src/System.Text.Json/src/System/Text/Json/JsonDocument.Parse.cs

```diff
@@ -107,12 +111,12 @@ public static JsonDocument Parse(ReadOnlyMemory<char> json, JsonReaderOptions re
             CheckSupportedOptions(readerOptions);
 
             ReadOnlySpan<char> jsonChars = json.Span;
-            int byteCount = Utf8JsonReader.Utf8Encoding.GetByteCount(jsonChars);
+            int byteCount = Utf8JsonReader.s_utf8Encoding.GetByteCount(jsonChars);
```

Comment:
nit: rename to `expectedByteCount` and `actualByteCount` or something to that affect rather than `byteCount` and `byteCount2`.